### ISSUE TITLE
libvecpf.so.1 is not created during cross compilation

### ISSUE THREAD

**Issue by tuliom** [Libvecpf/libvecpf#4]
Libvecpf is using ldconfig to create the SONAME symbolic link, causing errors during a cross compilation:

```
+Creating linker .so symlink libvecpf.so -> libvecpf-1.0.0.so
ln -f -s libvecpf-1.0.0.so libvecpf.so
+Running ldconfig to create the library compat symlink libvecpf.so.1 -> libvecpf-1.0.0.so
/sbin/ldconfig -l libvecpf-1.0.0.so
/sbin/ldconfig: libvecpf-1.0.0.so is for unknown machine 21.

/sbin/ldconfig: No link created since soname could not be found for libvecpf-1.0.0.so
```

It's necessary to get rid of ldconfig for this task as it isn't useful during cross compilations.

**Comment by ryanarn** [comment]
Hrm, I didn't consider building libvecpf with a cross-compiler when I created the Makefiles.

I don't think there's such a thing as a cross-ldconfig is there?

None-the-less, you should be able to accomplish the same thing by creating the symlink manually, just make sure you use DESTDIR relative paths in your symlink so that make install in DESTDIR doesn't give you an incorrect symlink.

**Comment by tuliom** [comment]
I already implemented the same thing on libdfp: https://github.com/libdfp/libdfp/commit/321011911b38dde55c79893f6e8540211c73a3dd

@ryanarn are there any restrictions against using libtool on these libraries?

**Comment by tuliom** [comment]
This issue has been fixed by commit 3ac475a3.
